src/style.css

```
@@ -1 +1,64 @@
-/* Styles go here */
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0 auto;
+  max-width: 1200px;
+}
```

You don't need to set max-width here. The header should be responsive
```suggestion
body {
  margin: 0 auto;
}
```